docs/basics.txt

```diff
@@ -599,19 +599,19 @@ graphs: {
 bin/gremlin-server.sh ./conf/gremlin-server/rest-gremlin-server.yaml
 ```
 +
-. The JanusGraph Server should now be running in REST mode and available for testing.  *curl* can be used to verify the server is working:
+. The JanusGraph Server should now be running in HTTP mode and available for testing.  *curl* can be used to verify the server is working:
 +
 ```
 curl -XPOST -Hcontent-type:application/json -d '{"gremlin":"g.V().count()"}' http://[IP for JanusGraph server host]:8182
 ```
 
 === Advanced JanusGraph Server Configurations
 
-==== WebSocket versus REST
+==== WebSocket versus HTTP
 
-JanusGraph Server must be run with a configuration for either WebSockets (for gremlin shell or other WebSocket programs) or re-configured for REST to work with REST clients.
+JanusGraph Server must be run with a configuration for either WebSockets (for gremlin shell or other WebSocket programs) or re-configured for HTTP to work with HTTP clients.
 
-There is, currently, no way to configure a single running instance of JanusGraph Server to communicate simultaneously with clients talking WebSockets and REST.  However, it is possible to create configuration files (following the steps in this chapter) for both WebSockets and REST and start two instances of the JanusGraph Server by pointing it to the various configurations when it is started.  In this way, the same JanusGraph database instance can be reached through different JanusGraph Servers through both WebSocket and REST calls. Make sure the port parameter in each yaml configuration is different if starting more than one JanusGraph Server on the same machine:
+There is, currently, no way to configure a single running instance of JanusGraph Server to communicate simultaneously with clients talking WebSockets and HTTP.  However, it is possible to create configuration files (following the steps in this chapter) for both WebSockets and HTTP and start two instances of the JanusGraph Server by pointing it to the various configurations when it is started.  In this way, the same JanusGraph database instance can be reached through different JanusGraph Servers through both WebSocket and HTTP calls. Make sure the port parameter in each yaml configuration is different if starting more than one JanusGraph Server on the same machine:
```

Comment:
Currently, there is no way...

Also since we are editing this paragraph lets put the sentences on different lines so it is easier to read diffs and update.